### PR TITLE
fix release action

### PR DESCRIPTION
**Pull request by stefanspiska**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -104,7 +134,7 @@ jobs:
           #update sdk to latest snapshot
           mvn versions:update-properties  -Dincludes=org.ehrbase.openehr.sdk:*  -DallowSnapshots=true
           #edit changelog
-          sed -i '5i ## [unreleased]\n ### Added \n ### Fixed \n' CHANGELOG.md
+          sed -i '7i ## [unreleased]\n ### Added\n ### Changed \n ### Fixed \n' CHANGELOG.md
```
Comment (HolgerReiseVSys) [inline .github/workflows/release.yml:137]: Should it be "8i" ?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -44,7 +69,12 @@ jobs:
           mvn versions:set -DnewVersion=${VERSION} -DprocessAllModules=true
           #Update sdk version latest release 
           mvn versions:update-properties  -Dincludes=org.ehrbase.openehr.sdk:*
+          sdk_version=$( grep -oPm1 "(?<=<ehrbase.sdk.version>)[^<]+" "bom/pom.xml")
+          echo ${sdk_version}
           #edit changelog
+          replace="0,/Changed/{s/Changed/Changed \n - Upgrade openEHR_SDK to version ${sdk_version} see https:\/\/github.com\/ehrbase\/openEHR_SDK\/blob\/develop\/CHANGELOG.md/}"
+          echo "${replace}"
```
Comment (HolgerReiseVSys) [inline .github/workflows/release.yml:76]: is the echo needed?

**Review by HolgerReiseVSys — COMMENTED** [review]
Please don't take this for a full review

File: .github/workflows/release.yml (hunk 1 above)
Comment (stefanspiska) [reply .github/workflows/release.yml:137]: no. I tested it 
https://github.com/stefanspiska/ehrbase/blob/develop/CHANGELOG.md?plain=1

Comment (HolgerReiseVSys) [reply .github/workflows/release.yml:136]: To me it looks like line 7 should be a blank line instead of line 17

Comment (vmueller-vg) [reply .github/workflows/release.yml:136]: The double blank line is not produced by this script.

Comment (vmueller-vg) [reply .github/workflows/release.yml:136]: Also @HolgerReiseVSys you are rigth it should be 8i if we assume line 7 to be blank. which actually produced the one time double blank line

**Review by vmueller-vg — APPROVED** [review]

**Review by vmueller-vg — APPROVED** [review]

**Pull request merged**